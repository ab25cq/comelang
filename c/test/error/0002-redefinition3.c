/*
PATTERN: x
*/

struct x {
	int x;
	int y;
	int x;
};
