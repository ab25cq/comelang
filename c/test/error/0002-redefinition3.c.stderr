field x duplicated in struct at test/error/0002-redefinition3.c:5:10
struct x {
         ^
